### PR TITLE
chore: speed up build time

### PR DESCRIPTION
**Pull request by amandakarina**

### CODE REVIEW

File: test/integration/appsource/cymbal_bank_test.go [hunk 1]
```diff
@@ -47,12 +47,12 @@ func TestSourceCymbalBank(t *testing.T) {
...
-		t.Parallel()
-		for appName, serviceNames := range testutils.ServicesNames {
-			appSourcePath := fmt.Sprintf("../../../6-appsource/%s", appName)
-			appFactory := tft.NewTFBlueprintTest(t, tft.WithTFDir(fmt.Sprintf("../../../3-appfactory/apps/%s", appName)))
-			for _, serviceName := range serviceNames {
+	for appName, serviceNames := range testutils.ServicesNames {
+		appSourcePath := fmt.Sprintf("../../../6-appsource/%s", appName)
+		appFactory := tft.NewTFBlueprintTest(t, tft.WithTFDir(fmt.Sprintf("../../../3-appfactory/apps/%s", appName)))
+		for _, serviceName := range serviceNames {
+			t.Run(fmt.Sprintf("%s/%s", appName, serviceName), func(t *testing.T) {
```
Comment (apeabody) [inline test/integration/appsource/cymbal_bank_test.go:54]: Probably need to capture the range variables

```suggestion
			serviceName := serviceName // capture range variable
			t.Run(fmt.Sprintf("%s/%s", appName, serviceName), func(t *testing.T) {
```

**Comment by apeabody** [comment]
Hmm, I didn't dig into this yet, but only the last test working in a parallel set can indicate a range variable that wasn't captured.

```
Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/frontend (590.60s)
Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-ledgerwriter (621.48s)
Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-balancereader (622.21s)
Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/accounts-contacts (651.06s)
Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/accounts-userservice (653.14s)
Step #32 - "appsource-verify":     --- PASS: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-transactionhistory (806.85s)
```

**Comment by apeabody** [comment]
> Hmm, I didn't dig into this yet, but only the last test working in a parallel set can indicate a range variable that wasn't captured.
> 
> ```
> Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/frontend (590.60s)
> Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-ledgerwriter (621.48s)
> Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-balancereader (622.21s)
> Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/accounts-contacts (651.06s)
> Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/accounts-userservice (653.14s)
> Step #32 - "appsource-verify":     --- PASS: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-transactionhistory (806.85s)
> ```

Looks like `transactionhistory` is always being used as the delivery-pipeline
`Step #32 - "appsource-verify": TestSourceCymbalBank/../../../6-appsource/cymbal-bank/accounts-contacts 2024-06-11T14:04:32Z command.go:100: Running command gcloud with args [deploy releases list --project=contacts-admin-52b8 --delivery-pipeline=transactionhistory --region=us-central1 --filter=name:c7ee59e --format json]`

**Comment by amandakarina** [comment]
> > Hmm, I didn't dig into this yet, but only the last test working in a parallel set can indicate a range variable that wasn't captured.
> > ```
> > Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/frontend (590.60s)
> > Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-ledgerwriter (621.48s)
> > Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-balancereader (622.21s)
> > Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/accounts-contacts (651.06s)
> > Step #32 - "appsource-verify":     --- FAIL: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/accounts-userservice (653.14s)
> > Step #32 - "appsource-verify":     --- PASS: TestSourceCymbalBank/../../../6-appsource/cymbal-bank/ledger-transactionhistory (806.85s)
> > ```
> 
> Yup, looks like `transactionhistory` (`suffixServiceName`) is always being used as the delivery-pipeline `Step #32 - "appsource-verify": TestSourceCymbalBank/../../../6-appsource/cymbal-bank/accounts-contacts 2024-06-11T14:04:32Z command.go:100: Running command gcloud with args [deploy releases list --project=contacts-admin-52b8 --delivery-pipeline=transactionhistory --region=us-central1 --filter=name:c7ee59e --format json]`

I captured in a map now.

**Review by apeabody — APPROVED** [review]
Thanks @amandakarina!

**Pull request merged**